uiowa/uiowa
# Image gallery2

Resolves https://github.com/uiowa/uiowa/issues/4827

How to test
blt ds --site=sandbox.uiowa.edu
https://sandbox.prod.drupal.uiowa.edu/single-image-gallery
Check to make sure that the node image styles is "medium - no crop" and the Photoswipe image is the original image.
